sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -31,17 +31,17 @@ class ArtifactManifestProperties(object):  # pylint: disable=too-many-instance-a
     :ivar bool can_read: Read Permissions for an artifact.
     :ivar bool can_list: List Permissions for an artifact.
     :ivar architecture: CPU Architecture of an artifact.
-    :vartype architecture: ~azure.containerregistry.ArtifactArchitecture
+    :vartype architecture: Optional[~azure.containerregistry.ArtifactArchitecture]
     :ivar created_on: Time and date an artifact was created.
-    :vartype created_on: ~datetime.datetime
-    :ivar str digest: Digest for the artifact.
+    :vartype created_on: Optional[~datetime.datetime]
+    :ivar Optional[str] digest: Digest for the artifact.
     :ivar last_updated_on: Time and date an artifact was last updated.
-    :vartype last_updated_on: ~datetime.datetime
+    :vartype last_updated_on: Optional[~datetime.datetime]
     :ivar operating_system: Operating system for the artifact.
-    :vartype operating_system: ~azure.containerregistry.ArtifactOperatingSystem
-    :ivar str repository_name: Repository name the artifact belongs to.
-    :ivar str size_in_bytes: Size of the artifact.
-    :ivar List[str] tags: Tags associated with a registry artifact.
+    :vartype operating_system: Optional[~azure.containerregistry.ArtifactOperatingSystem]
+    :ivar Optional[str] repository_name: Repository name the artifact belongs to.
+    :ivar Optional[int] size_in_bytes: Size of the artifact.
```

Comment:
Are you sure that this one is just a docs fix, and that the model itself is returning an `int`?

Reply:
1. The property type should be consistent in model return type and docstring.
2. AFAIK the doc portal is referenced by the docstring here. @kristapratico Can u help to confirm?

Reply:
Yes, the docs.microsoft.com ref docs will use these docstrings